cfy_manager/components/validations.py

```diff
@@ -151,13 +152,24 @@ def _validate_openssl_version():
 
 
 def _validate_inputs():
-    for key in (PRIVATE_IP, PUBLIC_IP):
-        ip = config[MANAGER].get(key)
+    Input = namedtuple('Input', 'key string flag')
+    required_inputs = [
+        Input(key=PRIVATE_IP, flag='--private-ip', string='Private IP'),
+        Input(key=PUBLIC_IP, flag='--public-ip', string='Public IP')
+    ]
+    for inp in required_inputs:
+        ip = config[MANAGER].get(inp.key)
```

Comment:
I like almost everything about this except calling this variable 'ip'

Maybe conf_input?

Reply:
Oh, definitely - this is a leftover from before. Fixing now